File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2956.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
        Scanner ler = new Scanner(System.in);
        double a = ler.nextDouble();
        double b = ler.nextDouble();
        System.out.printf("Concluimos que, dado o limite da entrada, a resposta seria:  y = f(x) = %.5f.\n",(a*b)/2);
    }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2956.cpp
#include <iostream>

using namespace std;

int main(){
    double a, b;
    cout.precision(5);
    cout.setf(ios::fixed);
    cin >> a >> b;
    cout << "Concluimos que, dado o limite da entrada, a resposta seria:  y = f(x) = " << (a*b)/2 << "." << endl;

    return 0;
}